scene/3d/physics/physics_body_3d.h

```diff
@@ -48,10 +48,10 @@ class PhysicsBody3D : public CollisionObject3D {
 
 	uint16_t locked_axis = 0;
 
-	Ref<KinematicCollision3D> _move(const Vector3 &p_motion, bool p_test_only = false, real_t p_margin = 0.001, bool p_recovery_as_collision = false, int p_max_collisions = 1);
+	Ref<KinematicCollision3D> _move(const Vector3 &p_motion, bool p_test_only = false, real_t p_margin = 0.001, bool p_recovery_as_collision = false, int p_max_collisions = 1, bool p_cancel_sliding = false);
 
 public:
-	bool move_and_collide(const PhysicsServer3D::MotionParameters &p_parameters, PhysicsServer3D::MotionResult &r_result, bool p_test_only = false, bool p_cancel_sliding = true);
+	bool move_and_collide(const PhysicsServer3D::MotionParameters &p_parameters, PhysicsServer3D::MotionResult &r_result, bool p_test_only = false, bool p_cancel_sliding = false);
```

Comment:
Why this change? There's no discrepancy here, why not keep both as `true` instead of changing one?

Reply:
Oh, is it me that changed that?

Reply:
Yep, it's true in both

Reply:
> Yep, it's true in both

Does this mean p_cancel_sliding was being applied all the time even prior to this change?